packages/main/src/Calendar.ts

```diff
@@ -404,7 +404,8 @@ class Calendar extends CalendarPart {
 		const uniqueSpecialDates: Array<SpecialCalendarDateT> = [];
 
 		validSpecialDates.forEach(date => {
-			const dateFromValue = UI5Date.getInstance(date.value);
+			const parsedDate = this.getFormat().parse(date.value) as Date | UI5Date;
+			const dateFromValue = UI5Date.getInstance(parsedDate);
```

Comment:
Why is that line needed? It does the same as 408. The return type of the parse method in DateFormat is Date | UI5Date, why do we need to create another instance?

Reply:
It seems like we don't need that line. Didn't saw that the return types are the same.